src/Native/Unix/System.Net.Security.Native/pal_gssapi.c

```diff
@@ -21,61 +21,61 @@
-static_assert(PAL_GSS_C_PROT_READY_FLAG == GSS_C_PROT_READY_FLAG, "");
-static_assert(PAL_GSS_C_TRANS_FLAG == GSS_C_TRANS_FLAG, "");
-static_assert(PAL_GSS_C_DCE_STYLE == GSS_C_DCE_STYLE, "");
-static_assert(PAL_GSS_C_IDENTIFY_FLAG == GSS_C_IDENTIFY_FLAG, "");
-static_assert(PAL_GSS_C_EXTENDED_ERROR_FLAG == GSS_C_EXTENDED_ERROR_FLAG, "");
-static_assert(PAL_GSS_C_DELEG_POLICY_FLAG == GSS_C_DELEG_POLICY_FLAG, "");
-
-static_assert(PAL_GSS_COMPLETE == GSS_S_COMPLETE, "");
-static_assert(PAL_GSS_CONTINUE_NEEDED == GSS_S_CONTINUE_NEEDED, "");
+c_static_assert(PAL_GSS_C_DELEG_FLAG == GSS_C_DELEG_FLAG);
+c_static_assert(PAL_GSS_C_MUTUAL_FLAG == GSS_C_MUTUAL_FLAG);
+c_static_assert(PAL_GSS_C_REPLAY_FLAG == GSS_C_REPLAY_FLAG);
+c_static_assert(PAL_GSS_C_SEQUENCE_FLAG == GSS_C_SEQUENCE_FLAG);
+c_static_assert(PAL_GSS_C_CONF_FLAG == GSS_C_CONF_FLAG);
+c_static_assert(PAL_GSS_C_INTEG_FLAG == GSS_C_INTEG_FLAG);
+c_static_assert(PAL_GSS_C_ANON_FLAG == GSS_C_ANON_FLAG);
+c_static_assert(PAL_GSS_C_PROT_READY_FLAG == GSS_C_PROT_READY_FLAG);
+c_static_assert(PAL_GSS_C_TRANS_FLAG == GSS_C_TRANS_FLAG);
+c_static_assert(PAL_GSS_C_DCE_STYLE == GSS_C_DCE_STYLE);
+c_static_assert(PAL_GSS_C_IDENTIFY_FLAG == GSS_C_IDENTIFY_FLAG);
+c_static_assert(PAL_GSS_C_EXTENDED_ERROR_FLAG == GSS_C_EXTENDED_ERROR_FLAG);
+c_static_assert(PAL_GSS_C_DELEG_POLICY_FLAG == GSS_C_DELEG_POLICY_FLAG);
+
+c_static_assert(PAL_GSS_COMPLETE == GSS_S_COMPLETE);
+c_static_assert(PAL_GSS_CONTINUE_NEEDED == GSS_S_CONTINUE_NEEDED);
 
 #if !HAVE_GSS_SPNEGO_MECHANISM
 static char gss_spnego_oid_value[] = "\x2b\x06\x01\x05\x05\x02"; // Binary representation of SPNEGO Oid (RFC 4178)
 static gss_OID_desc gss_mech_spnego_OID_desc = {.length = ARRAY_SIZE(gss_spnego_oid_value) - 1,
-                                                .elements = static_cast<void*>(gss_spnego_oid_value)};
+                                                .elements = gss_spnego_oid_value};
 static char gss_ntlm_oid_value[] =
     "\x2b\x06\x01\x04\x01\x82\x37\x02\x02\x0a"; // Binary representation of NTLM OID
                                                 // (https://msdn.microsoft.com/en-us/library/cc236636.aspx)
 static gss_OID_desc gss_mech_ntlm_OID_desc = {.length = ARRAY_SIZE(gss_ntlm_oid_value) - 1,
-                                              .elements = static_cast<void*>(gss_ntlm_oid_value)};
+                                              .elements = gss_ntlm_oid_value};
 #endif
 
 // transfers ownership of the underlying data from gssBuffer to PAL_GssBuffer
 static void NetSecurityNative_MoveBuffer(gss_buffer_t gssBuffer, struct PAL_GssBuffer* targetBuffer)
 {
-    assert(gssBuffer != nullptr);
-    assert(targetBuffer != nullptr);
+    assert(gssBuffer != NULL);
+    assert(targetBuffer != NULL);
 
-    targetBuffer->length = static_cast<uint64_t>(gssBuffer->length);
-    targetBuffer->data = static_cast<uint8_t*>(gssBuffer->value);
+    targetBuffer->length = (uint64_t)(gssBuffer->length);
```

Comment:
No need for last set of parens on each line.

Reply:
For readability I would keep them.